### PR TITLE
Update 4.1.2 notification and title

### PR DESCRIPTION
**Pull request by WendyYes**

### CODE REVIEW

**Comment by PRMerger4** [comment]
@WendyYes : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by WendyYes** [comment]
# sign-off

**Comment by PRMerger20** [comment]
@WendyYes : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by Jak-MS** [comment]
@chugugrace 

Can you review this PR?

IMPORTANT: When this content is ready to merge, you must add `#sign-off` in a comment or the approval may get overlooked.


#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

**Comment by PRMerger18** [comment]
@WendyYes : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger12** [comment]
@WendyYes : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger10** [comment]
@WendyYes : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger13** [comment]
@WendyYes : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger12** [comment]
@WendyYes : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Review by chugugrace — APPROVED** [review]

**Comment by chugugrace** [comment]
#sign-off

File: docs/ssdt/release-notes-ssdt-2019.md [hunk 1]
```diff
@@ -12,22 +12,31 @@ ms.custom: seo-lt-2019
...
+
+- **Due to a limitation of VS marketplace, the version 4.1.2 does not introduce new binaries to download. Version 4.1 contains the latest binaries.**
+
+- Since version 3.3, Power Query Source for SQL Server 2017 and Microsoft Oracle Connector for SQL Server 2019 have been excluded from the installation of this product. To continue using these two components, manually download and install them by yourselves. Here are the download links: [Power Query Source for SQL Server 2017 and 2019](https://www.microsoft.com/download/details.aspx?id=100619), [Microsoft Oracle Connector for SQL Server 2019](https://www.microsoft.com/download/details.aspx?id=58228)
 
 ## Common Issues
+
+- **If you met error during installation, you can check the logs under %temp%\SsdtisSetup.** Also, you can zip the folder and send the logs to ssistoolsfeedbacks@microsoft.com for troubleshooting.
```
Comment (Jak-MS) [inline docs/ssdt/release-notes-ssdt-2019.md:34]:
```suggestion
- **If you get an error during installation, you can check the logs under %temp%\SsdtisSetup.** Also, you can zip the folder and send the logs to ssistoolsfeedbacks@microsoft.com for troubleshooting.
```

**Comment by Jak-MS** [comment]
@WendyYes @chugugrace - Change to a TOC.yml file are not usually allowed via the public repos. This change appears to be straightforward, but please check the live site to make sure it's working correctly. In the future use the private repo for any changes to a TOC file.  

Thanks.

> Read about [authors working in the public repo](https://review.docs.microsoft.com/help/contribute/contribute-authors-in-public-repo?branch=main) for more information. 
>
> If you need help getting set up to work in the private repo, see [Set up Git repository locally for documentation](https://review.docs.microsoft.com/help/contribute/contribute-get-started-setup-local?branch=main).

**Pull request merged**